app/client/survey_admin/survey_admin_details.coffee

```diff
@@ -20,26 +21,29 @@ Template.survey_admin_details.onCreated ->
               formQuestions.push questionProps
             formProps.questions = formQuestions
             instance.forms.insert formProps
+            instance.active.set true
```

Comment:
Would this make the default state of the survey active as soon as it's created?  If so - I think we want to keep it deactivated until the admin has had a chance to set everything up and is ready to have it viewed by the public